skin.json

```diff
@@ -33,6 +44,19 @@
     "snapwikiskin": ["i18n"]
   },
   "ResourceModules": {
+    "skins.snapwikiskin.search": {
+      "dependencies": ["mediawiki.Uri", "wvui"],
```

Comment:
Where is ``mediawiki.Uri`` being used?

Reply:
Not sure, but I don't want to break anything